### DISCUSSION TITLE
Maintain constant temeprature

### DISCUSSION THREAD

**Discussion by boshk0** [Rem0o/FanControl.Releases#403] [Q&A]
I was trying the various curve types to lock the temperature of my GPU regardless of the load and environment temperature.
So fan I have somewhat good result with the graph curve, but I can only set it up to be accurate only at the current environment/load.

Is there better way to do it? I want to set 60 C for example the FanControl to control the RPM to maintain this temperature.

**Comment by Rem0o** [comment]
I don't have an "Auto" fan curve just yet. I've been working on it on and off, but it's really hard to get it to a state that is satisfying for a release.
